### ISSUE TITLE
Implement ES6 features 

### ISSUE THREAD

**Issue by bogdaner2** [GitbookIO/javascript#110]
I think, that`s a good idea to add information about some useful ECMAScript 6 features, such as: template literals/string interpolation, arrow function, modules etc.

**Comment by taherabbasm** [comment]
yeah that's a great idea, ES6 concepts should be included as mainly arrow function is highly used these days.

**Comment by munnokd** [comment]
Can you assign this issue to me? I will add this to the root directory.
